### ISSUE TITLE
Could you provide pre-trained weights for baseline architecture?

### ISSUE THREAD

**Issue by erobic** [cdancette/rubi.bootstrap.pytorch#5]
Would it be possible to provide the checkpoint for baseline architecture as well?

I got very low accuracy (only 21.74) with ```baseline.yaml```:

```
[S 2019-08-22 13:44:22] ...trap/engines/engine.py.120: eval_epoch.epoch: 11
[S 2019-08-22 13:44:22] ...trap/engines/engine.py.120: eval_epoch.loss: 4.28
[S 2019-08-22 13:44:22] ...trap/engines/engine.py.120: eval_epoch.accuracy_top1: 21.74
[S 2019-08-22 13:44:22] ...trap/engines/engine.py.120: eval_epoch.accuracy_top5: 70.43
```

which is very puzzling.

**Comment by erobic** [comment]
Instead of looking at the accuracies printed, I just had to use a proper evaluation script: https://github.com/cdancette/rubi.bootstrap.pytorch/issues/4
